tests/lib/rules/capitalized-comments.js

```diff
@@ -59,8 +59,8 @@ ruleTester.run("capitalized-comments", rule, {
         // No options: eslint/istanbul/jshint/jscs/globals?/exported are okay
```

Comment:
I think in this and the other rule tests, the intent was to check if directive comments were handled correctly.

Rather than changing all directive comments to non-directive comments, could we instead just turn off reporting of unused directives in those files?

Reply:
Looks like the old rule tests aren't actually failing now, because FlatRuleTester has its own `baseConfig`, where `linterOptions.reportUnusedDisableDirectives` isn't set.

https://github.com/eslint/eslint/blob/d1018fc5e59db0495aa4a7f501c9d3f831981f35/lib/rule-tester/flat-rule-tester.js#L538-L586

Now the question is shall we leave it that way?

Reply:
Good points. I reverted the rule tests.

On the one hand, it seems like rule tests should generally use the default config? But on the other hand, reporting disable directives seems like it would be unnecessary and even annoying during rule tests, so I'd probably prefer to leave that option off...

Reply:
I think reporting unused directives is probably best left off for rule tests -- I can't imagine people wanting those warnings intermixed with rule errors.